### ISSUE TITLE
provide json response with more information

### ISSUE THREAD

**Issue by fluxsaas** [lbeder/health-monitor-rails#8]
Hey,

i like your gem, very neat and extendable!

Would it be interesting for you, to provide a some more information in the controller:

https://github.com/lbeder/health-monitor-rails/blob/master/app/controllers/health_monitor/health_controller.rb#L14

e.g.:

````json
{
  "sidekiq": {
    "message": "latency 420.94790720939636 is greater than 30",
    "status": "FAILED",
    "timestamp": "2011-11-10T15:10:13+01:00"
  },
  "dalli": {
    "message": "",
    "status": "OK",
    "timestamp": "2011-11-10T15:10:13+01:00"
  }
}
````

**Comment by lbeder** [comment]
That's a good idea. I'll give it a try.

**Comment by lbeder** [comment]
Released as [v4.0.0](https://github.com/lbeder/health-monitor-rails/releases/tag/v4.0.0).

**Comment by fluxsaas** [comment]
awesome!
